javascript/index.js

```diff
@@ -1,22 +1,331 @@
+    liveLink: '#',
+    srcLink: '#',
+  },
+  {
+    projectName: 'Keeping track of hundreds of components',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s',
+    imgSrc: '#',
+    imgSrcAlt: '#',
+    tech1: 'HTML',
+    tech2: 'CSS',
+    tech3: 'Javascript',
+    tech4: 'Ruby on rails',
+    liveLink: '#',
+    srcLink: '#',
+  },
+
+];
+
+const populateMain = () => {
+  variables.sampleJobs.innerHTML = ` <div class="header-2-container">
+                  <h2 class="header header-2">My Recent Works</h2>
+                  </div>
+                  <div class="indicator">
+                  <img src="#" alt="" id="indicator">
+                  </div>
+                  <article class="cards">
+                  <div class="sample-containers">
+                      <div class="sample-img-container">
+                          <img src=${details[0].imgSrc} alt="" class="sample-img" id="sample-img-1">
+                      </div>
+                      <div class="sample-img-txt-head-container">
+                          <h3 class="sample-img-head-txt">${details[0].projectName}</h3>
+                      </div>
+                      <ul class="code-lang-container">
+                          <li class="ruby-on-rails">${details[0].tech4}</li>
+                          <li class="css">${details[0].tech3}</li>
+                          <li class="js">${details[0].tech2}</li>
+                          <li class="html">${details[0].tech1}</li>
+                      </ul>
+                      <div class="btn-container">
+                      <button type="button" class="btn" data-card='card0' 'card1'>See Project</button>
+                      </div>
+                  </div>
+                  <div class="sample-containers">
+                  <div class="sample-img-container">
+                      <img src=${details[1].imgSrc} alt="" class="sample-img" id="sample-img-1">
+                  </div>
+                  <div class="sample-img-txt-head-container">
+                      <h3 class="sample-img-head-txt">${details[1].projectName}</h3>
+                  </div>
+                  <ul class="code-lang-container">
+                      <li class="ruby-on-rails">${details[1].tech4}</li>
+                      <li class="css">${details[1].tech3}</li>
+                      <li class="js">${details[1].tech2}</li>
+                      <li class="html">${details[1].tech1}</li>
+                  </ul>
+                  <div class="btn-container">
+                      <button type="button" class="btn" data-card = 'card1'>See Project</button>
+                  </div>
+              </div>
+              <div class="sample-containers">
+              <div class="sample-img-container">
+                  <img src=${details[2].imgSrc} alt="" class="sample-img" id="sample-img-1">
+              </div>
```

Comment:
OPTIONAL

- [ ] Javascript makes it easier to do this kind of DOM. Using a loop instead of doing it manually would prove to be a better  coding practice